code/object/collideshipweapon.cpp

```diff
@@ -23,11 +23,14 @@
 #include "scripting/api/objs/model.h"
 #include "scripting/api/objs/vecmath.h"
 #include "playerman/player.h"
+#include "prop/prop.h"
 #include "ship/ship.h"
 #include "ship/shipfx.h"
 #include "ship/shiphit.h"
 #include "weapon/weapon.h"
 
+// Need a version of this for PROPS
```

Comment:
Can remove this

Reply:
Heh, this is why I left comments all over the place. I went through to remove the ones that ended up not being needed and found a few that could be finalized now that Props are working. Should be all cleaned up now.